stable/insights-agent/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.24.5
```

Comment:
also needs to bump the chart definitions for the agent

Reply:
Yea forgot to commit 🤦